style.css

```diff
@@ -0,0 +1,1498 @@
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "ARS Maquette", sans-serif;
+  src: url(assets/font/ARSMaquettePro-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Avalon";
+  src: url(assets/font/Avalon-Bold.ttf);
+}
+
+@font-face {
+  font-family: "Shentox", sans-serif;
+  src: url(assets/font/Shentox.ttf);
+}
+
+@font-face {
+  font-family: "ARS Maquette-bold";
+  src: url(assets/font/ARSMaquettePro-Bold.ttf);
+}
+
+body {
+  margin: 0 auto;
+  font-family: "ARS Maquette", sans-serif;
+}
+
+header a {
+  text-decoration: none;
+}
+
+/* Generic Classes */
+
+#test {
+  overflow: hidden;
+  text-indent: 100%;
+  white-space: nowrap;
+}
+
+.test {
+  overflow: hidden;
+  text-indent: 100%;
+  white-space: nowrap;
+}
+
+.white-text {
+  color: white;
+}
+
+.strikethrough {
+  text-decoration: line-through;
+}
+
+.gray-text {
+  color: #fff9;
+}
+
+.light-gray-text {
+  color: #aaa;
+}
+
+.uppercase {
+  text-transform: uppercase;
+}
+
+.smaller {
+  font-size: 13px;
+}
+
+.small {
+  font-size: 14px;
+}
+
+.gray {
+  background-color: gray;
+}
+
+.mr-30 {
+  margin-right: 30px;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.justify-content-center {
+  justify-content: center;
+}
+
+.align-items-center {
+  align-items: center;
+}
+
+.align-items-baseline {
+  align-items: baseline;
+}
+
+.align-items-flex-end {
+  align-items: flex-end;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.space-around {
+  justify-content: space-around;
+}
+
+.flex-end {
+  justify-content: flex-end;
+}
+
+.grid {
+  display: grid;
+}
+
+.bigger {
+  font-size: 56px;
+}
+
+/* Nav Section */
+
+.hamburger-icon-mobile {
+  display: none;
+}
+
+.search-icon-mobile {
+  display: none;
+}
+
+.tablet-tnw-logo {
+  display: none;
+}
+
+.latest-funding a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.footer-links-social a {
+  background-color: #333;
+  padding: 12px;
+  border-radius: 50%;
+  margin: 10px;
+  display: flex;
+}
+
+.footer-links-menu a {
+  color: #aaa;
+  margin: 20px 6px;
+  text-decoration: none;
+  font-size: 12px;
+  font-family: "Shentox", serif;
+}
+
+.latest-news h2 a {
+  text-decoration: none;
+  font-family: "Avalon", serif;
+  font-size: 28px;
+  color: #f42;
+}
+
+.card-1-mobile-text h3 a {
+  color: black;
+  text-decoration: none;
+}
+
+.latest-funding h2 a {
+  text-decoration: none;
+  font-size: 28px;
+  color: #f42;
+  font-family: "Avalon", sans-serif;
+}
+
+.card-3 li a {
+  color: #555;
+  font-size: 13px;
+}
+
+.latest-deals h2 a {
+  font-family: "Avalon", sans-serif;
+  font-size: 28px;
+  text-decoration: none;
+  color: #f42;
+}
+
+.footer-links-social li a {
+  border: transparent 1px solid;
+}
+
+.card-3 a:last-child {
+  text-decoration: none;
+  color: #aaa;
+}
+
+.footer-links-menu a:hover {
+  color: white;
+}
+
+.card-3 li a:hover {
+  color: black;
+}
+
+.latest-funding h2 a:hover {
+  color: black;
+}
+
+.latest-news h2 a:hover {
+  color: black;
+}
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color:black;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.black-color {
  color:black;
}
```
**Html file**

```HTML
<div class="black-color ">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_